Makefile.PL

```diff
@@ -35,7 +35,7 @@ my %WriteMakefileArgs = (
     "Test::More" => "0.94",
     "Test::NoWarnings" => 0
   },
-  "VERSION" => "0.17",
+  "VERSION" => "0.19",
```

Comment:
Why up to 0.19; what happened to 0.18?

Reply:
Another release was done to resolve an issue that came up during integration.
https://trello.com/c/5u93ZuVV